step-templates/sentry-release-tracking-token-auth.json

```diff
@@ -0,0 +1,53 @@
+{
+  "Id": "b40d5528-5de0-422d-a746-1e93cd04dea0",
+  "Name": "Sentry Release Tracking (Token Auth)",
+  "Description": "Posts a new release to Sentry, It can optionaly resolve all previous issues.\n\nIf the release already exists, it only applies the resolving.",
```

Comment:
Can you update the description to (briefly) explain the difference between this template and the previous one? 

Just an idea, but you could even link to the old one here stating that it doesn't work for new sentry accounts which cannot generate the legacy API key.